apps/balancer-tools/src/app/poolsimulator/analysis/page.tsx

```diff
@@ -21,6 +21,24 @@ export default function Page() {
             })
           : null}
       </div>
+      <div>
+        {initialData.tokens.map((token, index) => {
+          return (
+            <div className="flex items-center gap-x-4">
+              <span className="text-slate12">index</span>
+              <span className="text-slate12">{index}</span>
+              <span className="text-slate12">Symbol</span>
+              <span className="text-slate12">{token.symbol}</span>
+              <span className="text-slate12">Balance</span>
+              <span className="text-slate12">{token.balance}</span>
+              <span className="text-slate12">Rate</span>
+              <span className="text-slate12">{token.rate}</span>
+              <span className="text-slate12">Weight</span>
+              <span className="text-slate12">{token.weight}</span>
```

Comment:
I think we are never requesting the token weight

```
export const poolById = gql`
  query Pool($poolId: ID!) {
    pool(id: $poolId) {
      address
      owner
      poolType
      symbol
      swapFee
      amp
      c
      s
      alpha
      beta
      lambda
      tauAlphaX
      tauAlphaY
      tauBetaX
      tauBetaY
      u
      v
      w
      z
      dSq
      tokens {
        address
        symbol
        balance
        decimals
        priceRate
      }
    }
  }
`;
```

Reply:
For now yes (but I think that I should remove this weights for now)